extension/src/DocumentFunctions.ts

```diff
@@ -22,3 +22,25 @@ export async function searchTextFile(DocumentUri: vscode.Uri, StartPosition: num
 
     return { foundNode: (foundOffset >= 0), foundAtPosition: foundOffset };
 }
+
+export function getLineEnding(document: vscode.TextDocument) {
+    if (document.eol === vscode.EndOfLine.CRLF) {
```

Comment:
I don't really see the need for both functions but it could make for prettier code. No need for code cloning tho :)
 
```suggestion
return eolToLineEnding(document.eol);
```

Reply:
I assume I was quite drunk there... ;)